docs/administration/configuration/configuring-interactive-sessions/index.md

```diff
@@ -0,0 +1,53 @@
+# Configuring interactive sessions
+
+## Interactive sessions cleanup
```

Comment:
What do you think about dropping this intermediate section?

Reply:
I think it would be useful especially if in the future we add more sections, unrelated to the closure of interactive sessions.
I reworded it:
```
## Inactive interactive sessions cleanup
   ### Automatic closure
   ### Manual closure
```

In my opinion it gives more structure to the documentation page and to the table of contents - do you agree?